Project.toml

```diff
@@ -28,12 +29,13 @@ DelayEmbeddings = "2.6"
 Distances = "0.9, 0.10"
 FFTW = "1"
 Neighborhood = "0.2.4"
-QuadGK = "2"
+QuadGK = "2.9"
```

Comment:
Why is QuadGK still in the dependencies?

Reply:
It is used to compute another integral that is not the normal cdc for the Schurmann estimator, so can't remove it.